CONTRIBUTING.md

```diff
@@ -19,6 +19,75 @@ The Valkey project is led by a Technical Steering Committee, whose responsibilit
+[Clang Format Check/clang-format-check] ⭐ Run Main Check for formatting changes
+[Clang Format Check/clang-format-check]   🐳  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/3.sh] user= workdir=
+| Code is not formatted correctly. Here is the diff:
+| diff --git a/src/util.c b/src/util.c
+| index 8bd9a286c..2ea9fe384 100644
+| --- a/src/util.c
+| +++ b/src/util.c
+| @@ -110,7 +110,11 @@ static int stringmatchlen_impl(const char *pattern,
+|                      if (pattern[0] == string[0]) match = 1;
+|                  } else if (pattern[0] == ']') {
+|                      break;
+| -                } else if (patternLen == 0) { pattern--; patternLen++; break; } else if (patternLen >= 3 && pattern[1] == '-') {
+| +                } else if (patternLen == 0) {
+| +                    pattern--;
+| +                    patternLen++;
+| +                    break;
+| +                } else if (patternLen >= 3 && pattern[1] == '-') {
+|                      int start = pattern[0];
+|                      int end = pattern[2];
+|                      int c = string[0];
+[Clang Format Check/clang-format-check]   ❌  Failure - Main Check for formatting changes
+[Clang Format Check/clang-format-check] exitcode '1': failure
+[Clang Format Check/clang-format-check] 🏁  Job failed
+```
+
+### Here is a Successful Run
+
+```sh
+[Clang Format Check/clang-format-check]   ✅  Success - Main Set up Clang
+[Clang Format Check/clang-format-check] ⭐ Run Main Run clang-format
+[Clang Format Check/clang-format-check]   🐳  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/clang-format.sh] user= workdir=
+[Clang Format Check/clang-format-check]   ✅  Success - Main Run clang-format
+[Clang Format Check/clang-format-check] Cleaning up container for job clang-format-check
+[Clang Format Check/clang-format-check] 🏁  Job succeeded
+```
+```
+
```

Comment:
We should probably keep the DCO at the top as this is the most important information IMO for any new contributor. Also while you are at it, can you add some git commands to show how to add the DCO? I noticed that the commands were not obvious to some folks in the past.